packages/http-client-csharp/generator/Microsoft.Generator.CSharp.Input/src/InputTypes/Serialization/TypeSpecInputModelPropertyConverter.cs

```diff
@@ -30,23 +30,23 @@ private static InputModelProperty ReadInputModelProperty(ref Utf8JsonReader read
             string? description = null;
             InputType? propertyType = null;
             bool isReadOnly = false;
-            bool isRequired = false;
+            bool isOptional = false;
             bool isDiscriminator = false;
             IReadOnlyList<InputDecoratorInfo>? decorators = null;
             IReadOnlyList<string>? flattenedNames = null;
 
             while (reader.TokenType != JsonTokenType.EndObject)
             {
                 var isKnownProperty = reader.TryReadReferenceId(ref isFirstProperty, ref id)
-                    || reader.TryReadString(nameof(InputModelProperty.Name), ref name)
-                    || reader.TryReadString(nameof(InputModelProperty.SerializedName), ref serializedName)
-                    || reader.TryReadString(nameof(InputModelProperty.Description), ref description)
-                    || reader.TryReadWithConverter(nameof(InputModelProperty.Type), options, ref propertyType)
-                    || reader.TryReadBoolean(nameof(InputModelProperty.IsReadOnly), ref isReadOnly)
-                    || reader.TryReadBoolean(nameof(InputModelProperty.IsRequired), ref isRequired)
-                    || reader.TryReadBoolean(nameof(InputModelProperty.IsDiscriminator), ref isDiscriminator)
-                    || reader.TryReadWithConverter(nameof(InputModelProperty.Decorators), options, ref decorators)
-                    || reader.TryReadWithConverter(nameof(InputModelProperty.FlattenedNames), options, ref flattenedNames);
+                    || reader.TryReadString("Name", ref name)
```

Comment:
no need to change them now or you want it to be easier to change to lower case later?

Reply:
yeah it would be cumbersome to change it to lower case if I revert all these changes.
And after the alignment, some property's name changed such as `IsDiscriminator => Discriminator`